server/management/appointments/appointments.php

```diff
@@ -90,6 +99,28 @@ function rescheduleAppointment($appointmentId, $appointmentTimeId) {
 	echo json_encode($response);
 }
 
+function sendEmailConfirmation($appointmentId) {
+	GLOBAL $DB_CONN;
+	
+	try {
+		$query = "SELECT email FROM Appointment
+			JOIN Client ON Appointment.clientId = Client.clientId
+			WHERE appointmentId = ?";
+		$stmt = $DB_CONN->prepare($query);
+		$success = $stmt->execute(array($appointmentId));
+
+		if ($success == false) return;
+
+		$data = $stmt->fetch();
+		if (!isset($data) || !isset($data['email'])) return;
+
+		$confirmationMessage = AppointmentConfirmationUtilities::generateAppointmentConfirmation($appointmentId);
+		EmailUtilities::sendHtmlFormattedEmail($data['email'], 'VITA Appointment Rescheduled', $confirmationMessage, 'noreply@vita.unl.edu');
```

Comment:
My gut says to put `noreply@vita.unl.edu` in a config file. Additionally, if `noreply@vita.unl.edu` is the default value in the method arguments, I see no reason to include it here explicitly.

Reply:
oooh good call, putting it in the config file is a great idea! And again, yah I'll remove that since it's the default